src/output-json.c

```diff
@@ -501,6 +514,20 @@ OutputCtx *OutputJsonInitCtx(ConfNode *conf)
                 }
             }
 
+			const char *cee_format = ConfNodeLookupChildValue(conf, "cee-format");
+			if (cee_format != NULL)
+			{
+				if (strcasecmp(cee_format, "yes") == 0)
```

Comment:
Thanks for pointing this out, I must have missed this function. I'm still new to the code base. I saw something similar to what I did here in app-layer-parser.c line 281, so I followed suit. I will use this function from here on out.